plugins/modules/roles.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, John McCall (@lowlydba)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: roles
```

Comment:
Keeping the module names singular will fit better with the existing ones.

Reply:
Done

Reply:
Changed to role_member to allow a module that modifies the actual roles in the future.